### DISCUSSION TITLE
예약페이지로 넘어가지 않습니다. ㅠㅠ

### DISCUSSION THREAD

**Discussion by katyalex** [changdoc/naver-vaccine-macro#235] [General]
![image](https://user-images.githubusercontent.com/44580299/128456992-cf3cebef-e18e-4465-a1f9-89b39198898f.png)

안녕하세요. 매크로는 정상적으로 돌아가는데 브라우저에서 본인인증을 누르고 절차를 진행해도 예약페이지로 넘어가질 않네요... 혹시 해결방법이 있을까요? 정말 감사드립니다. 

핸드폰에서 예약페이지로 잘 넘어가긴 한데 PC에서 안 됩니다.

**Comment by changdoc** [comment]
매크로가 동작중인 상태라 페이지가 계속 새로고침 상태로 보이는데요.
매크로를 정지 시키시고 본인 인증 시도 해보시겠어요?

**Reply by katyalex** [reply]
네, 답장주셔서 정말 감사드립니다! 동작을 정지시켜도 해보았는데 크롬에서 안 되더라고요. 급하게 웨일로 넘어갔는데 웨일에서 예약페이지로 잘 넘어가더라고요! 크롬에서 왜 안 되는 건지 모르겠네요  ㅋㅋㅋㅋ 우선 오늘은 매크로를 돌려도 실패! 그래도 유용한 툴을 만들어주셔서 감사드립니다!
